### PR TITLE
Profile improvement

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
improve ux. fix some minor bugs

### CODE REVIEW

File: src/components/Header/web3/SelectWallet.tsx [hunk 1]
```diff
@@ -163,17 +163,16 @@ function Web3StatusInner() {
                 text={
                   <Text style={{ fontSize: '12px', textAlign: 'left', whiteSpace: 'normal' }}>
                     <Trans>
-                      You are not signed in with this wallet address. If you wish, you can
-                      <Text
-                        as="span"
+                      You are not signed in with this wallet address. If you wish, you can{' '}
+                      <span
```
Comment (namgold) [inline src/components/Header/web3/SelectWallet.tsx:167]: any reason for this?

File: src/components/Announcement/index.tsx [hunk 2]
```diff
@@ -226,8 +226,9 @@ export default function AnnouncementComponent() {
   useEffect(() => {
     if (userInfo?.identityId) {
       invalidateTag(ANNOUNCEMENT_TAGS)
+      refreshAnnouncement()
```
Comment (namgold) [inline src/components/Announcement/index.tsx:229]: consider to add cancel mechanism

File: src/components/Header/web3/SelectWallet.tsx (hunk 1 above)
Comment (nguyenhoaidanh) [reply src/components/Header/web3/SelectWallet.tsx:167]: dont know why, maybe some default style of <Text as='span'>

before
image
==========
after
image

File: src/components/Announcement/index.tsx (hunk 2 above)
Comment (nguyenhoaidanh) [reply src/components/Announcement/index.tsx:229]: I think it's ok for now.

**Review by namgold — APPROVED** [review]

**Pull request merged**